code/game/machinery/autolathe.dm

```diff
@@ -467,3 +454,4 @@
 		for(var/datum/design/D in files.known_designs)
 			if("hacked" in D.category)
 				files.known_designs -= D.id
+	SStgui.close_uis(src) // forces all connected users to re-open the TGUI, thus adding/removing hacked entries from lists
```

Comment:
Same here

Reply:
update_uis(src) does not reload static data. Only closing the entire window and re-opening it does.